app/__init__.py

```diff
@@ -5,27 +5,39 @@
 from peewee import *
 import datetime
 from playhouse.shortcuts import model_to_dict
+import werkzeug
 
 load_dotenv()
 app = Flask(__name__)
 
 # Change this open path depending on localhost OS folder structure/location
-pathCENTOS = "/root/GitHub/mlh-fellowship-project/app/static/data.json"
+
+# Tyler - I added these os paths to get them to work on my machine (or any other user's machine), if 
+# they don't work for whatever reason, just delete them if you like and keep the one you had already
+
+portfolio_dir = os.path.dirname(os.path.realpath(__file__))
+pathCENTOS = os.path.join(portfolio_dir, "static/data.json")
```

Comment:
I had no Idea about this join() method, super useful stuff and makes this process may more modular, thank you!